frontend/app/.server/locales/protected-fr.json

```diff
@@ -101,11 +101,20 @@
     "required-request": "Type de demande est requis."
   },
   "current-name": {
-    "page-title": "Nom actuel",
-    "page-description": "Les champs relatifs au nom actuel sont pré-remplis à partir du document d'identité principal. Vérifiez-les et mettez-les à jour avec le(s) nom(s) actuel(s) du demandeur si nécessaire. Ces noms seront utilisés pour leur NAS.",
-    "first-name": "Prénom(s) actuel(s)",
-    "middle-name": "Deuxième(s) prénom(s) actuel(s)",
-    "last-name": "Nom de famille actuel",
+    "page-title": "Nom pour le NAS",
+    "nom-enregistré": {
+      "description": "Le nom suivant est enregistré sur le principal document d'identité du demandeur :",
+      "first-name": "Prénom(s) : ",
+      "middle-name": "Deuxième prénom(s) : ",
```

Comment:
Remove the trailing spaces here and add them to the .tsx file (if it's actually required).